pep-0432.txt

```diff
@@ -6,13 +6,48 @@ Author: Nick Coghlan <ncoghlan@gmail.com>,
         Victor Stinner <vstinner@python.org>,
         Eric Snow <ericsnowcurrently@gmail.com>
 Discussions-To: capi-sig@python.org
-Status: Draft
+Status: Withdrawn
 Type: Standards Track
 Content-Type: text/x-rst
 Requires: 587
 Created: 28-Dec-2012
-Python-Version: 3.9
-Post-History: 28-Dec-2012, 2-Jan-2013, 30-Mar-2019
+Post-History: 28-Dec-2012, 2-Jan-2013, 30-Mar-2019, 28-Jun-2020
+
+
+PEP Withdrawal
+==============
+
+From late 2012 to mid 2020, this PEP provided general background and specific
+concrete proposals for making the CPython startup sequence easier to maintain
+and the CPython runtime easier to embed as part of a larger application.
+
+For most of that time, the changes were maintained either in a separate feature
+branch, or else as underscore-prefixed private APIs in the main CPython repo.
+
+In 2019, PEP 587 migrated a subset of those API changes to the public CPython
+API for Python 3.8+.
```

Comment:
Hum, maybe make it more explicit that PEP 587 is mostly the concrete implementation of the "configuration" part of this PEP.